File: app/src/main/java/com/vwoom/timelapsegallery/widget/WidgetGridRemoteViewsService.java
package com.vwoom.timelapsegallery.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class WidgetGridRemoteViewsService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new WidgetGridRemoteViewsFactory(this.getApplicationContext(), intent);
    }
}

File: app/src/main/java/com/vwoom/timelapsegallery/widget/WidgetGridRemoteViewsService.kt
package com.vwoom.timelapsegallery.widget

import android.content.Intent
import android.widget.RemoteViewsService
import com.vwoom.timelapsegallery.utils.InjectorUtils

class WidgetGridRemoteViewsService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return InjectorUtils.provideWidgetGridRemoteViewsFactory(this.applicationContext, intent)
    }
}